PublishParasoftResults/CoverageReportService.ts

```diff
@@ -108,22 +108,22 @@ export class CoverageReportService {
         packageToMerge.classes.forEach((classToMerge) => {
             const baseClass = basePackage.classes.get(classToMerge.classId);
             if (baseClass) {
-                this.mergeCoberturaClass(baseClass, classToMerge, packageToMerge.name);
+                this.mergeCoberturaClass(baseClass, classToMerge);
             } else {
                 basePackage.classes.set(classToMerge.classId, classToMerge);
             }
         });
     }
 
-    private mergeCoberturaClass = (baseClass: CoberturaClass, classToMerge: CoberturaClass, packageName: string): void => {
+    private mergeCoberturaClass = (baseClass: CoberturaClass, classToMerge: CoberturaClass): void => {
         this.sortLines(baseClass);
         this.sortLines(classToMerge);
         if (this.areClassesTheSame(baseClass, classToMerge)) {
             for (let i = 0; i < baseClass.lines.length; i++) {
                 baseClass.lines[i].hits += classToMerge.lines[i].hits;
             }
         } else {
-            throw new Error(`a conflict occurred while merging Class '${baseClass.fileName}' in '${packageName}'`);
+            throw new Error(`an inconsistent set of lines reported for class '${baseClass.fileName}'`);
```

Comment:
Should we use `name` instead of `fileName` here? 
Or if we keep using `fileName`, maybe it should be: "an inconsistent set of lines reported for class file '${baseClass.fileName}'